DataFormats/L1TParticleFlow/interface/PFTau.h

```diff
@@ -26,14 +26,16 @@ namespace l1t {
     PFTau() {}
     enum { unidentified = 0, oneprong = 1, oneprongpi0 = 2, threeprong = 3 };
     PFTau(const LorentzVector& p,
+          float iVector[80],
```

Comment:
What is this array storing?  I don't think the contents are set anywhere?  Can it be removed?